tools/downloader/pytorch_to_onnx.py

```diff
@@ -22,6 +22,21 @@ def positive_int_arg(values):
     return result
 
 
+def model_parameters(parameters):
+    if not parameters:
+        return dict()
+    params = {}
+    for element in parameters.split(','):
+        param, value = element.split('=')
+        try:
+            value = eval(value, {}, {})
+        except:
+            pass
```

Comment:
What's the purpose of this? If the evaluation fails, it's an invalid expression. You shouldn't just ignore it.